qemu/tests/cfg/win_virtio_driver_update_test.cfg

```diff
@@ -189,8 +189,23 @@
             device_hwid = '"PCI\VEN_1AF4&DEV_1052"'
             inputs = input1
             input_dev_bus_type_input1 = virtio
+            del usb_devices
+            move_rate = 80
+            move_duration = 1
+            btns = "left right middle side extra"
+            scrolls = "wheel-up wheel-down"
```

Comment:
see https://github.com/autotest/tp-qemu/pull/2042#discussion_r382550601

Reply:
updated the code, thanks~